### DISCUSSION TITLE
Using Strapi

### DISCUSSION THREAD

**Discussion by brookslybrand** [remix-austin/remixaustin-com#126] [Ideas]
@PaulBratslavsky gave a great talk at last night's meetup (recording to come) about using Remix and [Strapi](https://strapi.io/)

@sethdavis512 was talking after about the idea of us using Strapi as our CMS to drive some of the content on this site.

This is definitely interesting, and I wanted to start generating some conversation around this. So some questions
- Would we need to setup the free hosting? Would this be easy to do with Fly? Would Strapi support an account for us since this is for a community/open source meetup?
- What would we want to use Strapi for? Blog is one idea, but is there any other sort of content we may want to have?
- How do we actually want to go about building this? A little bit at a time during the meetups? A bunch of open tickets people can pick up. Build in public via YouTube streams? I'm pretty open to whatever

**Comment by sethdavis512** [comment]
> Would we need to setup the free hosting?

I'm going to assume yes. I don't think our hosting needs will be too much...I don't mind inquiring about free tiers for organizaions like ours. Maybe if we include a "hosted on Fly.io" link and a big logo they'd spot us 🤷🏻 

> Would this be easy to do with Fly?

After a quick search, looks like theres a community lead guide on how to host Strapi on Fly.io: https://forum.strapi.io/t/fly-io-deployment/22438

I'm wondering if we did the hosting process in one of our Meetups and we recorded it - posting the demo to our YouTube channel could be a nice contrib to the community.

> Would Strapi support an account for us since this is for a community/open source meetup?

Yeah I think this is more of a question for Paul 😛 

> What would we want to use Strapi for? Blog is one idea, but is there any other sort of content we may want to have?

Workshop content/guides. Meetup member stories (how they got into web dev). Blog posts from Remix Conf. Posted these ideas on a GH project board: https://github.com/orgs/remix-austin/projects/2

> How do we actually want to go about building this? A little bit at a time during the meetups? A bunch of open tickets people can pick up. Build in public via YouTube streams?

I don't think there's a one size fits all solution for this one. Maybe reserve 30mins at the end of a meetup to chat about GH issues/tickets.

**Reply by PaulBratslavsky** [reply]
We can self host Strapi in many ways, I typically use Digital Ocean, but there are many other options.  ( Render, Heroku, Amazon ). 

And it flexible for different types of content.

Doing a planning meeting at the Meetup I think would be a great start.

**Comment by sethdavis512** [comment]
Linking Paul's comment:
https://github.com/remix-austin/remixaustin-com/issues/127
